repos/spack_repo/builtin/packages/geomodel/package.py

```diff
@@ -93,6 +100,9 @@ class Geomodel(CMakePackage):
         depends_on("coin3d")
         depends_on("soqt")
         depends_on("gl")
+        depends_on("egl", when="@6.11:")
```

Comment:
I'm not sure if this is the right way to approach this. Since `egl` is only able to be provided as an external package, this would mean that `@6.11:` cannot be built anymore without external packages.

Tagging some maintainers for their input on this: @biddisco @snehring

Reply:
That is correct. egl is currently external only, so anyone who would want to install this would need to install whatever package implements egl in their ecosystem and configure it as an external.

Reply:
@stephenswat In that case, can you please change `.ci/gitlab/stacks/hep/spack.yaml` to use `~visualization`? Then these newer versions will be used in CI, which I think it preferred over continuously testing the last 6.10 version.